apps/xui/xui-webapp/sbox.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: xui-webapp
+spec:
+  values:
+    nodejs:
+      ingressHost: manage-case.sandbox.platform.hmcts.net
+      environment:
+        FEATURE_SECURE_COOKIE_ENABLED: false
+        SERVICES_MARKUP_API: http://em-npa-demo.service.core-compute-demo.internal
+        JURISDICTIONS: DIVORCE,PROBATE,FR,PUBLICLAW,IA,SSCS,ADOPTION,EMPLOYMENT,CIVIL,AUTOTEST1,CMC,HRS,BEFTA_JURISDICTION_1,PRIVATELAW,BEFTA_JURISDICTION_2,BEFTA_JURISDICTION_3,ST_CIC,ST_PHL,ST_CS,ST_SEND,ST_MH,BEFTA_MASTER
+        FEATURE_OIDC_ENABLED: true
+        SERVICES_IDAM_ISS_URL: https://forgerock-am.service.core-compute-idam-demo.internal:8443/openam/oauth2/realms/root/realms/hmcts
+        DEBUG: xuiNode:*
+        SERVICES_IDAM_OAUTH_CALLBACK_URL: oauth2/callback
+        STUB: false
+        SERVICES_CCD_CASE_ASSIGNMENT_API: http://aac-manage-case-assignment-demo.service.core-compute-demo.internal
+        SERVICES_PRD_API: http://rd-professional-api-demo.service.core-compute-demo.internal
+        SERVICES_ICP_API: http://em-icp-demo.service.core-compute-demo.internal
+        SERVICES_EM_DOCASSEMBLY_API: http://dg-docassembly-demo.service.core-compute-demo.internal
+        SERVICES_LOCATION_API: http://rd-location-ref-api-demo.service.core-compute-demo.internal
+        SERVICES_CASE_JUDICIAL_API: http://rd-judicial-api-demo.service.core-compute-demo.internal
+        SERVICES_HEARINGS_COMPONENT_API_SSCS: http://sscs-hearings-api-demo.service.core-compute-demo.internal
```

Comment:
Is this correct? Alot of URLs pointing to Demo

Reply:
Hi @thomast1906 ..its not, I was going to provisioned the HRs first then address all the env variables later once the pods are there